packages/api-server/api_server/repositories/tasks.py

```diff
@@ -1,5 +1,6 @@
 import sys
 from datetime import datetime
+from multiprocessing import Lock
```

Comment:
Using normal lock will likely cause a deadlock, need to use asyncio lock instead.

Reply:
https://github.com/open-rmf/rmf-web/pull/928/commits/cf37addbef64dcd00d6fd7bf0de7994682e4c651